hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/MessageContent.java

```diff
@@ -0,0 +1,59 @@
+package pl.allegro.tech.hermes.common.message;
+
+import org.apache.avro.Schema;
+import pl.allegro.tech.hermes.api.ContentType;
+import pl.allegro.tech.hermes.schema.CompiledSchema;
+
+import java.util.Optional;
+
+public class MessageContent {
+    private final ContentType contentType;
+    private final Optional<CompiledSchema<Schema>> schema;
+    private final byte[] data;
```

Comment:
everything is data! IMO `content` would fit perfectly as the name for actual content